plugins/scaffolder/src/next/TemplateWizardPage/Stepper/Stepper.tsx

```diff
@@ -102,6 +101,11 @@ export const Stepper = (props: StepperProps) => {
     setActiveStep(prevActiveStep => prevActiveStep - 1);
   };
 
+  const handleChange = useCallback(
+    (e: IChangeEvent) => setFormState(e.formData),
```

Comment:
If we have multiple steps - we don't end up replacing all previous data too do we, or do we need to merge here like we do on https://github.com/backstage/backstage/pull/15402/files#diff-b634e1da6a661c7e4d8e1205c15d5392edeb09c6eb37595cf4b0705d2e051b45R134

Reply:
@benjdlambert we don't, I'll add a good test to prove this

Reply:
@benjdlambert that is actually a good shout to merge.  I have updated to use the merge